src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -433,8 +436,9 @@ def __init__(self, pim_instance: Any | None = None):
             try:
                 from simple_upload_server.client import Client
 
+                token = os.getenv("UPLOAD_SERVER_TOKEN", "token")
```

Comment:
Please document new env var in doc\source\contributing\environment_variables.rst

Reply:
Done. Thank you.